lib/JSON/CalDAV/SubscriptionHandler.php

```
@@ -0,0 +1,122 @@
+            if ((int)$code > 299) {
+                $returncode = (int)$code;
+                break;
+            }
+        }
+
+        return [$returncode, null];
+    }
+
+    public function getSubscriptionInformation($nodePath, $node, $withRights) {
+        $baseUri = $this->server->getBaseUri();
+        $requestPath = $baseUri . $nodePath . '.json';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$requestPath'. (undefined)

(UnusedLocalVariable)